daemon/appinfo.c

```diff
@@ -1018,3 +1024,16 @@ appinfo_read_exec_dbus(appinfo_t *self, GKeyFile *ini, const gchar *group)
     }
     return exec;
 }
+
+/* ------------------------------------------------------------------------- *
+ * UTILS
+ * ------------------------------------------------------------------------- */
+static bool
+needs_exclusion_from_sandboxing(const gchar *exec)
+{
+    /* Alien apps and handlers must not be sandboxed */
```

Comment:
Would write "Apps and handlers launched via apkd-launcher must not be sandboxed"

Reply:
Changed it to what you suggested.